cid-redirects.json

```diff
@@ -1613,6 +1613,7 @@
   "/cid/10196": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source",
   "/cid/10122": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source",
   "/cid/10125": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-cyber-resilience-source",
+  "/cid/10127": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source",
```

Comment:
@amee-sumo, I changed the CID from "10126" to "10127" to resolve a merge conflict.